cri_lib/cri_controller.py

```diff
@@ -108,12 +113,12 @@ def connect(self, host: str, port: int = 3920) -> bool:
             return True
 
         except ConnectionRefusedError:
-            logging.error(
+            logger.error(
                 f"Connection refused: Unable to connect to {self.ip_address}:{self.port}"
             )
             return False
         except Exception as e:
-            logging.error(f"An error occurred: {str(e)}")
+            logger.error(f"An error occurred: {str(e)}")
```

Comment:
By using `logger.exception` the traceback of the exception is logged automatically.
```suggestion
            logger.exception("An error occurred while attempting to connect.")
```

Reply:
same like above removed f string